quilt/tools/build.py

```diff
@@ -143,15 +176,15 @@ def _file_to_spark_data_frame(ext, path, target, user_kwargs):
         dataframe = _file_to_data_frame(ext, path, target, user_kwargs)
     return dataframe
 
-def _file_to_data_frame(ext, path, target, user_kwargs):
+def _file_to_data_frame(ext, path, target, handler_args):
     _ = target  # TODO: why is this unused?
     logic = PARSERS.get(ext)
     the_module = importlib.import_module(logic['module'])
     if not isinstance(the_module, ModuleType):
         raise BuildException("Missing required module: %s." % logic['module'])
     # allow user to specify handler kwargs and override default kwargs
-    kwargs = dict(logic['kwargs'])
-    kwargs.update(user_kwargs)
+    kwargs = dict(logic['kwargs']).copy()
```

Comment:
Don't need `copy()`; `dict(...)` creates a new object.

That is, do either this:
`kwargs = dict(logic['kwargs'])`
or this:
`kwargs = logic['kwargs'].copy()`
I actually don't know if there's a difference, or which one is preferred.